spec/cucumber/formatter/json_spec.rb

```diff
@@ -550,8 +550,8 @@ module Formatter
             FEATURE
 
           define_steps do
-            Given(/^there are bananas$/) { data = 'YWJj'
-              embed data, 'mime-type;base64' }
+            Given(/^there are bananas$/) do data = 'YWJj'
```

Comment:
Like below, Given end typically are your wrappers, so I'd have them distinct. Move `end:554` to `555`?

Reply:
That's a weird one- don't know how that happened! Nice catch!

Reply:
For some weird reason the json_spec file seems _very_ fragile and without that extra line under my change, some of the tests following that one fail.